_FIPS201/authentication.md

```diff
@@ -127,6 +126,7 @@ Some of the characteristics of OCC-AUTH are as follows:
 - highly resistant to credential forgery
 - strong resistance to use of unaltered card by non-owner
 - usable with contact and contactless card readers
+- privacy-enhancing as the OCC data is never exported and is stored and processed directly on the PIV card 
```

Comment:
Agree, but want to clarify. The image is captured and processed off-card, while only the comparison of a template takes place on the card. Not convinced this is the best rewording.

```suggestion
- privacy-enhancing, as the biometric template is both never exported and compared directly on the PIV Card 
```